CONTRIBUTING.rst

```diff
@@ -89,7 +89,7 @@ tests, including testing other Python versions with tox::
 
     $ tox
 
-   To get flake8 and tox, just pip install them into your virtualenv. (Note that we use ``max-line-length = 100`` for flake8, this is configured in ``setup.cfg`` file.)
+   To get tox, just pip install them into your virtualenv. (Note that we use ``max-line-length = 119`` for flake8, this is configured in ``setup.cfg`` file.)
```

Comment:
What about not mentioning flake8 or any configuration file at all at this point?

```suggestion
   To get ``tox``, just ``pip install`` it into your virtualenv. In addition to tests, ``tox`` checks for code style and maximum line length (119 characters).
```